ferrocene/tools/self-test/src/targets.rs

```diff
@@ -10,12 +10,22 @@ use crate::linkers::Linker;
 use crate::report::Reporter;
 
 static SUPPORTED_TARGETS: &[TargetSpec] = &[
+    #[cfg(target_arch = "x86_64")]
```

Comment:
Why are these cfg'd?

Reply:
Otherwise the `aarch64` host tools will try to run the `x86_64` tests with the host CC, which doesn't work.